docker/django/Dockerfile

```diff
@@ -1,18 +1,45 @@
-FROM python:3.6
+ARG PYTHON_VERSION=3.6
+ARG IMAGE_VARIANT=slim
 
-ENV PYTHONUNBUFFERED 0
+# ==============================
+FROM helsinkitest/python:${PYTHON_VERSION}-${IMAGE_VARIANT} AS base_stage
+# ==============================
 
-RUN apt-get update \
-    && apt-get install --no-install-recommends -y gdal-bin python-gdal python3-gdal
+ENV PYTHONDONTWRITEBYTECODE true
+ENV PYTHONUNBUFFERED true
 
-COPY requirements.txt /usr/src/
+# Add tini init system https://github.com/krallin/tini
+ENV TINI_VERSION v0.18.0
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
+RUN chmod +x /tini
 
-RUN pip install --upgrade pip \
-    && pip install pip-tools \
-    && pip install --src /usr/src -r /usr/src/requirements.txt
+COPY --chown=appuser:appuser ./docker/django/docker-entrypoint.sh /app/
+ENTRYPOINT ["/tini", "--", "/app/docker/django/docker-entrypoint.sh"]
 
-RUN mkdir /code
+EXPOSE 8000
 
-WORKDIR /code
+# ==============================
+FROM base_stage AS development
+# ==============================
 
-COPY . /code
+RUN apt-install.sh gdal-bin build-essential
```

Comment:
don't forget to run `apt-cleanup.sh build-essential` during the same `RUN` layer to decrease the total image size 🙂

Reply:
Done in 4489697